src/IO/AbstractIOHandlerHelper.cpp

```diff
@@ -27,10 +27,39 @@
 #include "openPMD/IO/HDF5/HDF5IOHandler.hpp"
 #include "openPMD/IO/HDF5/ParallelHDF5IOHandler.hpp"
 #include "openPMD/IO/JSON/JSONIOHandler.hpp"
+
+#include <fstream>
 #include <nlohmann/json.hpp>
 
 namespace openPMD
 {
+namespace
+{
+    nlohmann::json
+    parseOptions( std::string const & options )
+    {
+        if( options.at( 0 ) == '@' )
+        {
+            std::fstream handle;
+            std::string filename = options;
+            filename.erase( 0, 1 );
+            handle.open( filename, std::ios_base::in );
```

Comment:
I think this is reasonable, but in practice (read: e.g. an ADIOS run at scale) libs like Ascent and AMReX with similar handlers of option files rather read from one rank and MPI-Broadcast the result to all ranks - avoiding to hammer the filesystem with many meta-data operations opening the same file.

Reply:
Ah, good catch. Will do.

Reply:
Done